diff --git a/package.json b/package.json
index 698043c..d9f0f44 100644
--- a/package.json
+++ b/package.json
@@ -70,6 +70,8 @@
     "redux-thunk": "^2.3.0"
   },
   "resolutions": {
+    "crypto-browserify": "https://registry.yarnpkg.com/@favware/skip-dependency/-/skip-dependency-1.1.1.tgz",
+    "http-signature": "https://registry.yarnpkg.com/@favware/skip-dependency/-/skip-dependency-1.1.1.tgz",
     "monaco-editor": "0.20.0",
     "rxjs": "6.6.3"
   },
diff --git a/webpack.config.js b/webpack.config.js
index ae72fd8..529d6bf 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -89,6 +89,10 @@ module.exports.getWebpackConfig = (config, options) => {
             ...config.module,
             rules: removeDataTestAttributeInProduction(options.production, excludeExtractionLoaderForMonaco(config.module.rules)),
         },
+        node: {
+            ...config.node,
+            crypto: false
+        },
         plugins: [
             ...updateForkTsCheckerPluginSettings(config.plugins),
             new MonacoWebpackPlugin({
